### PR TITLE
fix(Projects): Improved Task management UI

### PR DESCRIPTION
**Pull request by jakeaturner**

### CODE REVIEW

File: client/src/components/projects/ProjectView.jsx [hunk 1]
```diff
@@ -3233,30 +3257,38 @@ const ProjectView = (props) => {
...
+                  <div className='task-actions-div'>
+                  <Header sub>Actions</Header>
+                    <div className='flex-row-div left-flex'>
+                      <Popup
+                        key='edit-task'
+                        trigger={
+                          <Button
+                            size='tiny'
+                            icon='pencil'
+                            color='green'
```
Comment (ethanaturner) [inline client/src/components/projects/ProjectView.jsx:3269]: Change to blue for consistency

**Review by ethanaturner — CHANGES_REQUESTED** [review]
Besides color change, LGTM.

File: client/src/components/projects/ProjectView.jsx (hunk 1 above)
Comment (jakeaturner) [reply client/src/components/projects/ProjectView.jsx]: Fixed

**Review by ethanaturner — APPROVED** [review]
_ET_. 02/06/23.

**Pull request merged**